bench/FileConnectionBench.java

```diff
@@ -0,0 +1,48 @@
+package benchmark;
+
+import com.sun.cldchi.jvm.JVM;
+import com.sun.midp.crypto.SecureRandom;
+import java.io.OutputStream;
+import java.util.Random;
+import javax.microedition.io.Connector;
+import javax.microedition.io.file.FileConnection;
+import org.mozilla.MemorySampler;
+
+public class FileConnectionBench {
+  void runBenchmark() {
+    try {
+      long start, time;
+
+      start = JVM.monotonicTimeMillis();
+
+      String privateDir = System.getProperty("fileconn.dir.private");
+      Random random = new Random();
+      String filename = new javax.bluetooth.UUID(java.lang.Math.abs(random.nextInt())).toString();
```

Comment:
I originally did that, but I actually want to create unique files each time, so I can see if database growth over time affects memory usage. In 549e3e9 I switch to using *System.currentTimeMillis* to create unique-enough filenames.